docs/api/fexpr/__init__.rst

```diff
@@ -5,7 +5,7 @@
     __init__(self, e)
     --
 
-    Create a new :class:`dt.FExpr` object out of `e`.
+    Create a new :class:`dt.FExpr` object out of ``e``.
 
     The ``FExpr`` is serve as a simple wrapper of the underlying object,
```

Comment:
nitpick here on the sentence, maybe `The ``FExpr`` serves as a simple wrapper...`

Reply:
That's a grammatical error, so definitely not a nitpick :)